modules/redis-ai.ts

```diff
@@ -0,0 +1,266 @@
+import { Module } from './module.base'
+import * as Redis from 'ioredis';
+
+export class RedisAI extends Module {
+
+    /**
+     * Initializing the RedisAI object
+     * @param options The options of the Redis database.
+     * @param throwError If to throw an exception on error.
+     */
+    constructor(options: Redis.RedisOptions, throwError = true) {
+        super(options, throwError)
+    }
+
+    /**
+     * 
+     * @param key 
+     * @param type 
+     * @param data 
+     * @param shape 
+     */
+    async tensorset(key: string, type: TensorType, data?: string[] | Buffer[], shape?: string): Promise<void> {
+        try {
+            let args: (string | Buffer)[] = [key, type];
+            if(shape !== undefined)
+                args = args.concat(['shape', shape])
```

Comment:
<shape> is not a keyword. 
Example command:
```
redis> AI.TENSORSET mytensor FLOAT 2 2 VALUES 1 2 3 4
OK
```
Please check https://github.com/RedisAI/redisai-js/blob/master/src/tensor.ts#L51

Reply:
@filipecosta90 Thanks, I'll check it out

Reply:
@filipecosta90 Do you have any example for BLOB data maybe? can't' seem to get it.

Reply:
yep, here it is:
https://github.com/RedisAI/redisai-js/blob/master/tests/test_client.ts#L153